archives/2019-03-06/Overview.html

```diff
@@ -0,0 +1,8011 @@
+
+WebVTT files provide captions or subtitles for video content, and also text video descriptions <a data-link-type="biblio" href="#biblio-maur">[MAUR]</a>, chapters for content navigation, and more generally any form of metadata that is time-aligned with audio or video content.</p>
+
+<p>This specification is based on the <a href="https://w3c.github.io/webvtt/">Draft Community Group Report</a> of the
+<a href="https://www.w3.org/community/texttracks/">Web Media Text Tracks Community Group</a>.</p>
+
+  </div>
+  <h2 class="no-num no-toc no-ref heading settled" id="status"><span class="content">Status of this document</span></h2>
+  <div data-fill-with="status">
+
+<p> <em>This section describes the status of this document at the time of its publication. Other
+documents may supersede this document. A list of current W3C publications and the latest revision
+of this technical report can be found in the <a href="https://www.w3.org/TR/">W3C technical reports
+index at https://www.w3.org/TR/.</a></em> </p>
+
+<p>This document was produced by the <a href="https://www.w3.org/AudioVideo/TT/">
+<abbr title="World Wide Web Consortium">W3C</abbr> Timed Text Working Group</a> as a Candidate
+Recommendation. This document is intended to become a <abbr title="World Wide Web Consortium">W3C</abbr>
+Recommendation. If you wish to make comments regarding this document, please send them to
+<a href="mailto:public-tt@w3.org?subject=%5Bwebvtt%5D">public-tt@w3.org</a>
+(<a href="mailto:public-tt-request@w3.org?subject=subscribe">subscribe</a>,
+<a href="http://lists.w3.org/Archives/Public/public-tt/">archives</a>) with <code>[webvtt]</code>
+at the start of your email’s subject. All comments are welcome.
+<abbr title="World Wide Web Consortium">W3C</abbr> publishes a Candidate Recommendation to indicate
+that the document is believed to be stable and to encourage implementation by the developer
+community. This document will remain a Candidate Recommendation at least until 15 July 2018
+in order to ensure the opportunity for wide review.</p>
+
+<p>Please see the Working Group's <a href="https://www.w3.org/wiki/TimedText/WebVTT_Implementation_Report">Implementation Report</a>.</p>
+
+<p>For this specification to exit the CR stage, at least 2 independent implementations of every
+feature defined in this specification need to be documented in the implementation report. The
+implementation report is based on implementer-provided test results for the
+<a href="https://github.com/web-platform-tests/wpt/tree/master/webvtt">test suite</a>. The Working
+Group does not require that implementations are publicly available but encourages them to be so.</p>
+
+<div data-fill-with="at-risk">
+  <p>The following features are at-risk, and may be dropped during the CR period:</p>
+  <ul>
+    <li>collision avoidance with snap-to-lines false</li>
```

Comment:
I don't think it is clear exactly what feature is being marked as at risk here, since "collision avoidance" isn't defined anywhere. Can we be more specific, for example is this referring to §7.2 Processing cue settings, bullet 10, span-to-lines is false, bullet 4 :

> If there is a position to which the boxes in boxes can be moved while maintaining the relative positions of the boxes in boxes to each other such that none of the boxes in boxes would overlap any of the boxes in output, and all the boxes in boxes would be within the video’s rendering area, then move the boxes in boxes to the closest such position to their current position, and then jump to the step labeled done positioning below. If there are multiple such positions that are equidistant from their current position, use the highest one amongst them; if there are several at that height, then use the leftmost one amongst them.

?

Reply:
Yes, referencing that part. I can add the specific bullet, something like:
```
collision avoidance with snap-to-lines false, §7.2 Processing cue settings, bullet 10, span-to-lines is false, bullet 4.
```

I used this section for inspiration (https://www.w3.org/TR/selectors-4/#status) which was quite brief in the at-risk section.

Reply:
I'm not sure what is possible in respec, but could you add a fragment identifier to the relevant text and make the "collision avoidance" an `<a href="#collision-avoidance">collision avoidance</a>`?

Reply:
Given that I'm editing the raw HTML in this snapshot, I can definitely add a fragment identifier in the right place manually. Not sure how to go about making it generate one for the future, should we need one, maybe doesn't matter.

Reply:
I guess adding manually will be enough. The next iteration will either keep the text or removing it so the fragid won't be needed.

Reply:
Updated, new link https://raw.githack.com/w3c/webvtt/8108fd5699f510926b439c27e7ee2c52b118b8d6/archives/2019-03-06/Overview.html

Reply:
Thanks, that's enough to resolve this I think.